balrogscript/script.py

```diff
@@ -29,8 +29,11 @@ def create_locale_submitter(e, balrog_auth, config):
 
         complete_info = e['completeInfo']
         partial_info = e.get('partialInfo')
-        submitter = ReleaseSubmitterV9(api_root=config['api_root'], auth=auth,
-                                       dummy=config['dummy'])
+        submitter = ReleaseSubmitterV9(
+            api_root=config['api_root'], auth=auth,
+            dummy=config['dummy'],
+            suffix=e.get('blob_suffix', ''),
```

Comment:
I'm not certain if we set dummy anywhere -- but https://dxr.mozilla.org/build-central/source/tools/lib/python/balrog/submitter/cli.py?q=ReleaseSubmitterV9&redirect_type=single#591 shows that -dummy isn't used if we set a suffix...

Reply:
I've got a patch to build-tools to fix that.